wfe2/wfe.go

```diff
@@ -1030,55 +1048,59 @@ func (wfe *WebFrontEndImpl) Account(
 		return
 	}
 
-	// Only the Contact and Status fields of an account may be updated this way.
-	// For key updates clients should be using the key change endpoint.
-	var accountUpdateRequest struct {
-		Contact *[]string       `json:"contact"`
-		Status  core.AcmeStatus `json:"status"`
-	}
+	// If the body was not empty, then this is an account update request.
+	if string(body) != "" {
+		// Only the Contact and Status fields of an account may be updated this way.
```

Comment:
Can we break out the update logic into its own function.

Reply:
Sounds good to me. Resolved in d1b4407